app/templates/ratemypdf_stats.html

```diff
@@ -0,0 +1,262 @@
+    {% endif -%}
+  </h4>
+  {% if description | length > 0 -%}
+  <div class="collapse" id="{{label}}-info">
+    <p>{{ description | safe }}</p>
+  </div>
+  {% endif -%}
+  <div class="percentage-position-chart">
+    {% if val > mean + std_dev %}
+    <div class="percentage-position-chart__bar percentage-position-chart__bar--red" style="width: {{ val * 100 }}%;"></div>
+    {% elif val > mean - std_dev %}
+    <div class="percentage-position-chart__bar percentage-position-chart__bar--yellow" style="width: {{ val * 100 }}%;"></div>
+    {% else %}
+    <div class="percentage-position-chart__bar percentage-position-chart__bar--green" style="width: {{ val * 100 }}%;"></div>                  
+    {% endif %}
+    <div class="percentage-position-chart__arrow percentage-position-chart__arrow--red" style="left: {{ val * 100}}%;"></div>
+    <div class="percentage-position-chart__label percentage-position-chart__label--score">Score: {{ val }}%</div>
+    <div class="percentage-position-chart__label percentage-position-chart__label--median">Median: 10%</div>
+    <div class="percentage-position-chart__label percentage-position-chart__label--mean">Mean: 12%</div>
+    <div class="percentage-text">{{ (val * 100) | format_number }}%</div>
+  </div>
+</div>
+{% endmacro -%}
+{% macro score_position_widget(label, title, description, val, mean, std_dev) -%}
+<div class="mb-3">
+  <h4>{{ title }} 
+    {% if description | length > 0 -%}
+    <a href="#{{ label }}-info" data-bs-toggle="collapse" role="button" aria-expanded="false" aria-controls="{{ label }}-info"><i class="fas fa-plus"></i></a>
+    {% endif -%}
+  </h4>
+  {% if description | length > 0 -%}
+  <div class="collapse" id="{{ label }}-info">
+    <p>{{ description | safe }}</p>
+  </div>
+  {% endif -%}
+  <div class="score-position-chart">
+    {% if val > mean + std_dev %}
+    <div class="score-position-chart__bar score-position-chart__bar--red" style="width: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    <div class="score-position-chart__arrow score-position-chart__arrow--red" style="left: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    {% elif val > mean - std_dev %}
+    <div class="score-position-chart__bar score-position-chart__bar--yellow" style="width: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    <div class="score-position-chart__arrow score-position-chart__arrow--red" style="left: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>    
+    {% else %}
+    <div class="score-position-chart__bar score-position-chart__bar--green" style="width: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    <div class="score-position-chart__arrow score-position-chart__arrow--green" style="left: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    {% endif %}
+    <div class="score-position-chart__label score-position-chart__label--score">Score: 75</div>
```

Comment:
```suggestion
```

I'd remove Score; it's redundant with the below text

Reply:
it's not actually visible but agreed we only need one. will address this with the other thoughts from Monday's session